ext/intl/dateformat/dateformat.stub.php

```diff
@@ -136,7 +136,7 @@ public function getTimeZone(): IntlTimeZone|false {}
      * @tentative-return-type
      * @alias datefmt_set_timezone
      */
-    public function setTimeZone($timezone): ?bool {} // TODO return true on success
```

Comment:
I'm assuming this change should've gone to 8.1 and the rest to master?

Reply:
why ? it would not be really true for 8.1 and above ?

Reply:
Sorry, I looked at the diff wrong. This is right of course.